api/uniter/unit.go

```diff
@@ -221,19 +221,19 @@ func (u *Unit) WatchRelations() (watcher.StringsWatcher, error) {
 	return w, nil
 }
 
-// Service returns the service.
+// Application returns the unit's application.
```

Comment:
Thank you for changing the comment \o/ 
Previous one was not particularly insightful :D